src/crab/bitset_domain.hpp

```
@@ -16,7 +16,7 @@ class BitsetDomain final {
   public:
     BitsetDomain() { non_numerical_bytes.set(); }
 
-    explicit BitsetDomain(const bits_t& non_numerical_bytes) : non_numerical_bytes{non_numerical_bytes} {}
+    explicit BitsetDomain(bits_t non_numerical_bytes) : non_numerical_bytes{std::move(non_numerical_bytes)} {}
```

Clang (14.0.6)

[warning] constructor does not initialize these fields: non_numerical_bytes

(cppcoreguidelines-pro-type-member-init)